### PR TITLE
Adding campaign group tpl and refactoring to reuse campaign code

### LINKED ISSUE #1894: Campaign Group template
Preprocess vars and create a template for the Campaign Group template.  

Ideally figure out ways to re-use as much code as possible.

### PR DESCRIPTION
**Pull request by blisteringherb**
Fixes #1894

@DFurnes 
@aaronschachter 
@barryclark

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign_group/node--campaign_group.tpl.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      <div class="cta-wrapper">
+        <div class="cta">
+          <h3><?php print $call_to_action; ?></h3>
+          <div class="cta_button"><?php print $cta_link; ?></div>
+        </div>
+      </div>
+    <?php endif; ?>
+
+    <?php if (!empty($galleries)): ?>
+      <div class="gallery-wrapper">
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign_group/node--campaign_group.tpl.php:90]: Nothing we need now, but I could see us making this a themeable function so all this gallery markup is in one place.

**Comment by aaronschachter** [comment]
Same deal, as long as all is tested on campaigns and campaign_groups, ship it :+1: value added

**Comment by DFurnes** [comment]
Cool cool. :+1:

**Comment by barryclark** [comment]
@weerd too, he's going to be rocking front-end on this now

**Pull request merged**